### DISCUSSION TITLE
Want to know how to control the size of the output file

### DISCUSSION THREAD

**Discussion by LiuJiao0408** [Parcels-code/Parcels#1176] [General]
When_ I do a long simulation, the output file is often large.
This makes the system unable to create a large NC file (e.g. 22G)

I have tried this statement:

> outfile = ParticleFile(output_filename, pset, outputdt = out_interval，chunksizes=[200,1])

But this statement reports an aurgument error. do not have 'chunksizes'
I wonder if there is any good way (except to reduce the frequency of output)?

**Comment by philippemiron** [comment]
Duplicate of https://github.com/OceanParcels/parcels/discussions/1177.
